app/main.py

```diff
@@ -1,8 +1,21 @@
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_person = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
```

Comment:
You could simplify the check using `get`
```suggestion
        if person.get("wife"):
```
And save value to a variable for further use using the walrus operator
```suggestion
        if wife_name := person.get("wife"):
```

Reply:
![image](https://github.com/mate-academy/py-person-class/assets/148888194/671d9430-8e0f-4e09-8eff-23a62dc029d4)

Reply:
In boolean we have True, but have't Name.